### ISSUE TITLE
[Bug] Error generating notebook DD

### ISSUE THREAD

**Issue by Davidson-Mike** [DidierRLopes/GamestonkTerminal#409]
**Describe the bug**
```
---------------------------------------------------------------------------
Exception encountered at "In [12]":
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-12-09e82de1763f> in <module>
----> 1 mw.prepare_df_financials(ticker, "income")

~\Documents\PY-Projects\GamestonkTerminal\gamestonk_terminal\fundamental_analysis\market_watch_model.py in prepare_df_financials(ticker, statement, quarter)
     75     s_header_end_trend = ("5-year trend", "5- qtr trend")[quarter]
     76     df_financials = pd.DataFrame(
---> 77         columns=a_financials_header[0 : a_financials_header.index(s_header_end_trend)]
     78     )
     79

ValueError: '5-year trend' is not in list
```

**To Reproduce**
```
mill
dd -t HYLN
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Python version 3.6.8

**Comment by jmaslek** [comment]
If you look at the market watch page: (https://www.marketwatch.com/investing/stock/hyln/financials )  They don't have any financials there...  We can probably try catching an empty list @DidierRLopes

**Comment by jmaslek** [comment]
Closed #410
